### ISSUE TITLE
Use Schaefer as default to prevent error running the app outside of the docker image

### ISSUE THREAD

**Issue by Remi-Gau** [SIMEXP/giga_connectome#97]
Was trying to quickly see if the app functions as expected.

## steps to reproduce

- Install data
- Create a virtual env
- Install package

```bash
giga_connectome giga_connectome/data/test_data/ds000017-fmriprep22.0.1-downsampled-nosurface \
            /outputs \
            participant \
            -w /work \
            --participant_label 1 \
            --reindex-bids
```

## output

```
[11:29:33] INFO     {'bids_dir': PosixPath('giga_connectome/data/test_data/ds000017-fmriprep22.0.1-downsampled-nosurface'),              workflow.py:35
                    'output_dir': PosixPath('/outputs'), 'analysis_level': 'participant', 'participant_label': ['1'], 'work_dir':                      
                    PosixPath('/work'), 'atlas': 'MIST', 'denoise_strategy': 'simple', 'standardize': 'zscore', 'smoothing_fwhm': 5.0,                 
                    'reindex_bids': True, 'bids_filter_file': None, 'calculate_intranetwork_average_correlation': False, 'verbosity': 2}               
           INFO     {'name': 'MIST', 'parameters': {'atlas': 'BASC', 'template': 'MNI152NLin2009bAsym', 'resolution': '03', 'suffix':       atlas.py:49
                    'dseg'}, 'desc': [7, 12, 20, 36, 64, 122, 197, 325, 122, 444], 'templateflow_dir': None}                                           
Traceback (most recent call last):
  File "/home/remi/github/giga_connectome/env/bin/giga_connectome", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/remi/github/giga_connectome/env/lib/python3.11/site-packages/giga_connectome/run.py", line 127, in main
    workflow(args)
  File "/home/remi/github/giga_connectome/env/lib/python3.11/site-packages/giga_connectome/workflow.py", line 51, in workflow
    atlas = load_atlas_setting(args.atlas)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/remi/github/giga_connectome/env/lib/python3.11/site-packages/giga_connectome/atlas.py", line 69, in load_atlas_setting
    p = templateflow.api.get(
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/remi/github/giga_connectome/env/lib/python3.11/site-packages/templateflow/conf/__init__.py", line 65, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/remi/github/giga_connectome/env/lib/python3.11/site-packages/templateflow/api.py", line 112, in get
    raise Exception("No results found")
Exception: No results found
```

**Comment by htwangtw** [comment]
Ah this is because MIST is not included in templateflow default.
I did try to add MIST to templateflow but didn't follow up.
I used mist as default simply because that's the most used atlas in the lab

There are some clunky mechanism for supplying one's own atlas in templateflow format, and even worse documentation for it.
Easy solution: use a different atlas as default (schafer) 
Hard solution: package MIST in the installation process (I have no clue how to do this)
